airbyte-integrations/connectors/destination-s3-glue/src/main/java/io/airbyte/integrations/destination/s3_glue/GlueOperations.java

```diff
@@ -83,16 +84,23 @@ public void deleteTable(String databaseName, String tableName) {
 
   }
 
-  private Collection<Column> transformSchema(JsonNode jsonSchema, S3Format s3Format) {
-    if (jsonSchema.has("properties")) {
-      Map<String, JsonNode> properties = objectMapper.convertValue(jsonSchema.get("properties"), new TypeReference<>() {});
-      return properties.entrySet().stream()
-          .map(es -> new Column().withName(es.getKey()).withType(transformSchemaRecursive(es.getValue(), s3Format)))
-          .collect(Collectors.toSet());
-    } else {
-      return Collections.emptySet();
-    }
+private Collection<Column> transformSchema(JsonNode jsonSchema, S3Format s3Format) {
+  if (jsonSchema.has("properties")) {
+    Map<String, JsonNode> properties = objectMapper.convertValue(jsonSchema.get("properties"), new TypeReference<>() {});
+    return properties.entrySet().stream()
+        .map(es -> {
+          String key = es.getKey();
+          String stdName = AvroConstants.NAME_TRANSFORMER.getIdentifier(key);
+          if (!key.equals(stdName)) {
+            key = stdName;
+          }
```

Comment:
Can this be replaced with key = stdname?
or 
String key = AvroConstants.NAME_TRANSFORMER.getIdentifier(es.getKey());

This either changes nothing, or reassigns key